### PR TITLE
Add rubygem-hammer_cli_foreman_tasks package

### PR DESCRIPTION
**Pull request by ehelms**
Based off https://github.com/theforeman/foreman-packaging/pull/469
See also: https://github.com/theforeman/hammer-cli-foreman-tasks/pull/16

### CODE REVIEW

File: rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec [hunk 1]
```diff
@@ -0,0 +1,62 @@
+%global gem_name hammer_cli_foreman_docker
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec]: tasks

File: rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+%global confdir hammer
+
+Summary: Contains the code for showing of the tasks (results and progress) in the Hammer CLI
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec]: Should be less than 72 chars, just remove the "Contains the code" bit, it's superfluous

File: rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+%if 0%{?rhel} == 6
+Requires: ruby(abi)
+%else
+Requires: ruby(release)
+%endif
+
+Requires: ruby(rubygems)
+Requires: rubygem(hammer_cli_foreman) >= 0.1.2
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec]: should be >= 0.1.1 and < 0.2.0

File: rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+Requires: rubygem(powerbar)
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec]: >= 1.0.11 and < 1.1.0

File: rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec (hunk 2 above)
Comment (domcleal) [inline rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec]: The gem summary is better actually, although it contains a typo: "Foreman CLI plugin for showing tasks information for resoruces [resources!] and users"

File: rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec (hunk 1 above)
Comment (domcleal) [inline rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec]: Looks like this is the current cause of the build/test failure.

**Comment by domcleal** [comment]
Many thanks for resurrecting this!

**Comment by domcleal** [comment]
The package should install a configuration file in hammer's .d directory to activate the plugin on installation.  For now you can include this as a source, but it should also be added into the gem so it's maintained in one place (see also 1db3327 for discovery).

**Comment by ehelms** [comment]
@domcleal Updated with a proper release of the gem with the config file.

**Comment by domcleal** [comment]
Thanks @ehelms!  Will build into nightly and 1.8 plugin repos.

**Pull request merged**